assets/queries/cloudFormation/logging_is_enabled/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+CxPolicy [ result ] {
+    resource := input.document[i].Resources[name]
+    resource.Type == "AWS::CloudFront::Distribution"
+    
+    loggingEnabled := resource.Properties.DistributionConfig.Logging.Enabled
```

Comment:
Logging object doesn't seem to include an *Enabled* attribute as explained here https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-distribution-logging.html